internal/pkg/manifest/lb_web_svc.go

```diff
@@ -52,7 +52,7 @@ type LoadBalancedWebServiceConfig struct {
 	Logging          `yaml:"logging,flow"`
 	Sidecars         map[string]*SidecarConfig `yaml:"sidecars"` // NOTE: keep the pointers because `mergo` doesn't automatically deep merge map's value unless it's a pointer type.
 	Network          NetworkConfig             `yaml:"network"`
-	Publish          PublishConfig             `yaml:"publish"`
+	PublishConfig    PublishConfig             `yaml:"publish"`
```

Comment:
Why do we rename this field? After renaming this seems to be not consistent with the `Network` above.

Reply:
Oh it was because this field has the same name as the struct function `Publish()`, which caused problem during testing (an error like `Publish` is not a function). I should actually do the same for the other workloads.